index.html

```diff
@@ -225,6 +225,46 @@ <h4>Skills</h4>
         </div>
       </div>
     </section>
+    <section>
+      <div class="contact-container">
+        <h1 class="contact-heading">Contact me</h1>
+        <p class="contact-paragraph">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding, I'd love to help
+          with it.
+        </p>
+        <form class="contact-form" action="https://formspree.io/f/mjvdqdaj method="POST">
+          <input
+            class="contact-input"
+            required
+            maxlength="30"
+            name="name"
+            type="text"
+            placeholder="Please enter your name (max 30 characters)"
+          />
+          <input
+            class="contact-input"
+            required
+            name="email"
+            type="email"
+            pattern="[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}"
+            placeholder="Your Email address"
+          />
+          <textarea
+          class="contact-input"
+          required
+          maxlength="500"
+          name="message"
+          cols="30"
+          rows="9"
+          placeholder="Write your message here (up to 500 characters)"
+        ></textarea>
```

Comment:
Well! 👍 I noticed that you have added meaningful placeholder text in your form fields, which is great for providing guidance to users. However, upon reviewing the design, I found that the styles of your placeholder text do not match the design. 🎨 It is important to ensure that your placeholder text aligns with the design to maintain a consistent and visually appealing user experience. 🖌️ By doing so, you will enhance the overall aesthetics and cohesiveness of your form.  🙏 👍 

| Your | Professional |
| --- | --- |
|  ![image](https://github.com/the-faizmohammad/my-portfolio/assets/55840999/0c504873-3025-40d0-9c4e-33dd895f6624)  |   ![image](https://github.com/the-faizmohammad/my-portfolio/assets/55840999/aeca9b85-781c-4ed1-8786-5a98014eea4d)  |